Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocAutoCompleteReferenceValue.css

```diff
@@ -1,248 +1,146 @@
+  padding-right: calc(var(--default-icon-size) + var(--remotion-themed-spacing-small));
 }
 
-span.bocAutoCompleteReferenceValueButton img:hover
+.bocAutoCompleteReferenceValue span.content span.bocAutoCompleteReferenceValueInput input
 {
-  background-color: #DEECFC;
+  width: calc(var(--remotion-themed-control-width) + var(--default-icon-size) + var(--remotion-themed-spacing));
+  padding-left: var(--remotion-themed-spacing);
+  padding-right: calc(var(--default-icon-size) + var(--remotion-themed-spacing-small));
 }
 
-span.bocAutoCompleteReferenceValueButton img:active
+.bocAutoCompleteReferenceValue span.icon img
 {
-  background-color: #C5E0FC;
+  position: relative;
+  width: var(--default-icon-size);
+  height: var(--default-icon-size);
+  padding-left: var(--remotion-themed-spacing);
+  z-index: 1;
+  pointer-events: none;
 }
 
-span.bocAutoCompleteReferenceValue span.body
+.bocAutoCompleteReferenceValue input.error
 {
-  display: inline-block;
-  margin-top: 1px;
-  margin-left: 1px;
-  padding: 1px;
+  background-image: url('../Image/sprite.svg#AutoCompleteReferenceValueErrorIndicator');
+  background-size: var(--default-icon-size) var(--default-icon-size);
+  background-position: calc(var(--remotion-themed-control-width) - var(--default-icon-size)) center;
+  background-repeat: no-repeat;
 }
 
-span.bocAutoCompleteReferenceValue.disabled span.content,
-span.bocAutoCompleteReferenceValue .disabled span.content
+.bocAutoCompleteReferenceValue.disabled span.content,
+.bocAutoCompleteReferenceValue .disabled span.content
 {
-  color: gray;
+  color: var(--color-disabled-text);
   cursor: default;
 }
 
-span.bocAutoCompleteReferenceValue span.body span.content
-{
-  margin-right: 1px;
-}
-
-span.bocAutoCompleteReferenceValue.readOnly span.icon img
-{
-  margin-right: 6px;
-}
-
-span.bocAutoCompleteReferenceValue.disabled span.icon img,
-span.bocAutoCompleteReferenceValue .disabled span.icon img
+.bocAutoCompleteReferenceValue.readOnly span.content
 {
-  opacity: 0.4;
-  filter: progid:DXImageTransform.Microsoft.BasicImage(opacity=0.40),
-          progid:DXImageTransform.Microsoft.BasicImage(grayscale=1);
+  display: inline-flex;
+  align-items: center;
 }
 
-/* withoutOptionsMenu */
-span.bocAutoCompleteReferenceValue span.content.withoutOptionsMenu
+.bocAutoCompleteReferenceValue.readOnly span.content span
 {
-  right: 0;
+  padding-left: calc(var(--remotion-themed-spacing) + var(--remotion-themed-border-width));
 }
 
-/* hasOptionsMenu */
-span.bocAutoCompleteReferenceValue span.optionsMenu span.DropDownMenuSelect,
-span.bocAutoCompleteReferenceValue.disabled span.optionsMenu span.DropDownMenuSelect,
-span.bocAutoCompleteReferenceValue.readOnly span.optionsMenu span.DropDownMenuSelect
+.bocAutoCompleteReferenceValue.readOnly span.content.hasIcon span
 {
-  height: 1.5em;
-  top: 0;
+  padding-left: calc(var(--default-icon-size) + 2 * var(--remotion-themed-spacing) + var(--remotion-themed-border-width));
```

Comment:
Maybe introduce a new variable for `2 * var(--remotion-themed-spacing)`, you could name it `--remotion-themed-spacing-double`

Reply:
no, I just dropped the double variable. if we need 2x spacing because it's meant to be padding above+below, we should do this inline.